python/cudf/cudf/_lib/orc.pyx

```diff
@@ -133,8 +130,7 @@ cdef read_orc_args make_read_orc_args(filepath_or_buffer,
     cdef read_orc_args args = read_orc_args(
         make_source_info([filepath_or_buffer])
     )
-    args.stripe = stripe
-    args.stripe_count = stripe_count
+    args.stripe_list = stripes
```

Comment:
```suggestion
    args.stripes = stripes
```

Reply:
Thanks, Mark. I updated this in all appropriate places.